### ISSUE TITLE
Road navigation line isn't avaible sometimes

### ISSUE THREAD

**Issue by Markcortbass** [fransschreuder/libosmscout#38]
Road navigation line isn't avaible sometimes. 
![screenshot20151022_230909941](https://cloud.githubusercontent.com/assets/6825753/10690265/e4bfd52c-7980-11e5-8a32-a48a66de1934.png)

**Comment by fransschreuder** [comment]
true, this doesn't happen when you choose "north up". It's a rendering issue, looking into this.
